src/V3EmitMk.cpp

```diff
@@ -31,11 +31,557 @@ class EmitMk final {
 public:
     // METHODS
 
+    struct FileOrConcatenatedFilesList final {
+        std::string fileName{};
+        std::vector<std::string> concatenatedFileNames{};
+
+        // Concatenating file score for debugTestConcatenation
+        std::int64_t dbgScore = 0;
+
+        bool isConcatenatingFile() const { return !concatenatedFileNames.empty(); }
+    };
+
+    struct FileNameWithScore final {
+        std::string name;
+        std::int64_t score;
+    };
+
+    // Data of a single work unit used in `singleConcatenatedFilesList()`.
+    struct WorkList final {
+        std::int64_t totalScore{0};
```

Comment:
```suggestion
        std::int64_t totalScore = 0;
```

Only to match our typical style.